econplayground/main/models.py

```diff
@@ -474,3 +474,87 @@ class Meta:
 
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+class Question(models.Model):
+    adaptive = models.BooleanField(default=False)
+    ap_correct = models.ForeignKey(
+        'self',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='ap_continue'
+    )
+    ap_incorrect = models.ForeignKey(
+        'self',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='ap_intervene'
+    )
+    title = models.CharField(max_length=256, default='Untitled', unique=True)
+
+    embedded_media = models.TextField(blank=True, default='')
+    graph = models.ForeignKey(
+        Graph, on_delete=models.CASCADE, blank=True)
```

Comment:
I think this would need a `null=True` as well, if your intention is to have an optional Graph foreign key here.

Reply:
Sure thing